src/sentry/conf/server.py

```diff
@@ -1649,6 +1649,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "organizations:sdk-crash-detection": False,
     # Enables commenting on PRs from the Sentry comment bot.
     "organizations:pr-comment-bot": False,
+    # slack lookup
```

Comment:
I'd recommend putting something a bit more descriptive like "Enables alternate Slack channel lookup via scheduled message" here in case someone w/o context sees it.